pyLOM/vmmath/src/svd.c

```diff
@@ -1009,6 +1010,148 @@ int ztsqr_svd(dcomplex_t *Ui, double *S, dcomplex_t *VT, dcomplex_t *Ai, const i
 	return info;
 }
 
+int srandomized_svd(float *Ui, float *S, float *VT, float *Ai, const int m, const int n, const int r, const int q, MPI_Comm comm) {
+	/*
+		Randomized single value decomposition (SVD) with oversampling and power iterations with the algorithm from
+		
+		Erichson, N. B., Voronin, S., Brunton, S. L., & Kutz, J. N. (2016). Randomized matrix decompositions using R. arXiv preprint arXiv:1608.02148.
+
+		Ai(m,n)  data matrix dispersed on each processor.
+
+		Ui(m,n)  POD modes dispersed on each processor (must come preallocated).
+		S(n)     singular values.
+		VT(n,n)  right singular vectors (transposed).
+	*/
+	int info = 0;
+	int ii   = 0;
+	// Multiply per a random matrix
+	float *omega;
+	float *Y;
+	unsigned int seed = (unsigned int)time(NULL);
```

Comment:
I'd keep the seed outside and would set this up at the cython layer, to keep the C layer clean.

Reply:
Shall I put always the same number for reproducibility in the cython layer? or a random int each time?